src/signals/incident-management/containers/IncidentOverviewPage/saga.js

```diff
@@ -40,9 +42,47 @@ export function* openIncident(action) {
   yield put(push(navigateUrl));
 }
 
+export function* getFilters() {
+  const requestURL = `${CONFIGURATION.API_ROOT}signals/v1/private/me/filters/`;
+
+  try {
+    const result = yield authCall(requestURL);
+
+    yield put(getFiltersSuccess(result.results));
+  } catch (error) {
+    yield put(getFiltersFailed(error));
+  }
+}
```

Comment:
I don't see why filters are stored in the back-end AND sent along with the list fetch request. Am i missing something or can one of the methods get deleted?

Reply:
that is in the saga of the Filter container.

/src/signals/incident-management/containers/Filter/saga.js